ramls/samples/fieldName.sample

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Did you guys talk this structure through with UI guys, this is different from the transformations that are required by the mapping profile API. 
So when retrieving back they should be able to map these 2

Reply:
As far as I know, `referenceDataValue` is untranslatable. Yes, I showed the structure of response to Victor
I will clarify is it translatable with Magda

Reply:
discussed with Victor, all ok, going to merge